src/main/java/com/google/opensesame/user/UserEntity.java

```diff
@@ -35,8 +34,11 @@ public UserEntity(UserData userData) {
     interestTags = userData.getTags();
     email = userData.getEmail();
     userId = userData.getUserID();
-    projectIds = userData.getProjectIDs();
-    menteeIds = userData.getMenteeIDs();
+    projectIds =
+        new ArrayList<String>(
+            userData.getProjects().stream()
+                .map(x -> x.getRepositoryId())
```

Comment:
nit: Could we use a more descriptive name here?
```suggestion
                .map(projectData -> projectData.getRepositoryId())
```